INTERPRETING.md

```diff
@@ -4,6 +4,11 @@ All tests are declared as text files located within this project's `test`
 directory. In order to execute Test262 tests, runtimes must observe the
 following semantics.
 
+**Note** When these instructions change in any substantive way, the `version`
+property of the JSON-formatted `package.json` file will be incremented. In this
+way, consumers who are transitioning between revisions of Test262 can more
+easily determine the cause of new test failures.
```

Comment:
as a follow up, should we connect this to semver? How do we define major, minor and patches in this repo?

So far I'm happy with the latest versions, but we should explain a major tag refers to a breaking change in the required interpretation, as minors should only address new tags that can be interpreted.